src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class SortCommand extends Command {
```

Comment:
A header comment here would be helpful.

Reply:
So i should implement 1 pull request per feature added am i correct?
In this case i added 2 features for 1 pull request because they were in the same W5.11 section.